packages/parser/tests/unit/schema/vpc-latticev2.test.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Test built in schema
+ *
+ * @group unit/parser/schema/
+ */
+
+import { VpcLatticeV2Schema } from '../../../src/schemas/vpc-latticev2';
+import vpcLatticeV2Event from '../../events/vpcLatticeV2Event.json';
+import vpcLatticeEventV2PathTrailingSlash from '../../events/vpcLatticeEventV2PathTrailingSlash.json';
+
+describe('VpcLatticeV2 ', () => {
+  it('should parse VpcLatticeV2 event', () => {
+    const parsed = VpcLatticeV2Schema.parse(vpcLatticeV2Event);
+    expect(parsed).toEqual(vpcLatticeV2Event);
```

Comment:
How come this is one of the only tests where we assert on the parsed outputs?

Should we do the same elsewhere or is it redundant?

Reply:
My bad, missed a lot of tests. I have added all the other cases, please check if I missed anything.